common/persistence/shardManager_test.go

```diff
@@ -0,0 +1,399 @@
+// The MIT License (MIT)
```

Comment:
We are in the process of standardizing file names with snake_case. We update as we touch. Please rename this file and corresponding code file